lib/handlers/receiver_handler.rb

```diff
@@ -82,6 +85,10 @@ def initialize(
     # connects receiver client to SRCommonHandler#broker
     # and creates receiver
     def on_container_start(container)
+      # Prepare source options
+      source = {}
+      source[:address] = @broker.amqp_address
+      source[:filter] = @selector if @selector
```

Comment:
@rkubis Question is, do we want to do source = {}, or do it the way you do in PR #31 ?

Reply:
Hmm, good question.

Reply:
This looks better for me now, but some comments can be welcome.